src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +174,298 @@
         result     (dispatch [event-id {:amount amount}])]
     (testing "amount set"
       (is (match? amount (get-in result [:db :wallet :ui :send :amount]))))))
+
+(h/deftest-event :wallet/clean-send-data
+  [event-id dispatch]
+  (let [token-symbol      "ETH"
+        token             {:symbol   "ETH"
+                           :name     "Ether"
+                           :networks #{{:chain-id 421614}
+                                       {:chain-id 11155420}
+                                       {:chain-id 11155111}}}
+        receiver-networks [421614 11155420]]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send        {:token-display-name token-symbol
+                                   :token              token
+                                   :receiver-networks  receiver-networks}
+                     :other-props :value}}})
+    (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui]) :send)))))
+
+(h/deftest-event :wallet/select-address-tab
+  [event-id dispatch]
+  (let [expected-db {:wallet {:ui {:send {:select-address-tab "tab"}}}}]
+    (reset! rf-db/app-db
+      {:wallet {:ui {:send nil}}})
+    (is (match? expected-db (:db (dispatch [event-id "tab"]))))))
+
+(h/deftest-event :wallet/clean-send-address
+  [event-id dispatch]
+  (reset! rf-db/app-db
+    {:wallet {:ui {:send {:to-address  "0x01"
+                          :recipient   {:recipient-type :saved-address
+                                        :label          "label"}
+                          :other-props :value}}}})
+  (is (not (contains? (get-in (dispatch [event-id]) [:db :wallet :ui :send]) :to-address)))
```

Comment:
`(get-in (dispatch [event-id]) [:db :wallet :ui :send])` can be put into `let` to avoid its repetition